src/storage.ts

```diff
@@ -529,6 +529,8 @@ export class Storage extends Service {
    *     Force the use of the User Project metadata field to assign operational
    *     costs when an operation is made on a Bucket and its objects.
    * @property {boolean} [standard=true] Specify the storage class as Standard.
+   * @property {string} [storageClass] The new storage class. (`standard`,
```

Comment:
I believe we need a complete list from here: https://github.com/googleapis/nodejs-storage/blob/a1135afdce9daf44eea6d7ff8c82d541e413a661/src/storage.ts#L635-L641

Reply:
Done!